modules/layers/src/scatterplot-layer/scatterplot-layer-vertex.glsl.js

```diff
@@ -47,6 +47,8 @@ varying vec2 unitPosition;
 varying float innerUnitRadius;
 
 void main(void) {
+  DECKGL_MAIN_START();
```

Comment:
@tsherif I find it pretty awkward to have to add this to every layer. Can I define module injection into standard locations (`vs:main-start` and such) without registering my own hook?

Reply:
I suppose we could make those strings options for the `shaderHook` argument to `setModuleInjection`. Would that be enough?